pkgs/development/python-modules/streamz/default.nix

```diff
@@ -38,6 +39,9 @@ buildPythonPackage rec {
     requests
   ];
 
+  # zict dependency is now python 3.6+ only
```

Comment:
Wording is a little confusing, implies that it would support e.g. py3.5 if not for zict. Maybe just say is python 3.6+ only, though that's implied by ``disabled=pythonOlder 3.6``